packages/contracts-bedrock/test/properties/PROPERTIES.md

```diff
@@ -108,7 +109,7 @@ As another layer of defense, the following properties are defined which assume b
 It’s worth noting that these properties will not hold for a live system
 
 | id  | milestone           | description                                                                                                                        | halmos | medusa |
-| --- | ---                 | ---                                                                                                                                | ---    | ---    |
+| --- | ------------------- | ---------------------------------------------------------------------------------------------------------------------------------- | ------ | ------ |
```

Comment:
what do you use to format markdown? so I can use it as well and keep things consistent :sparkling_heart: